package/src/skia/core/Image.ts

```diff
@@ -6,5 +6,5 @@ import { useRawData } from "./Data";
 /**
  * Returns a Skia Image object
  * */
-export const useImage = (source: DataSource) =>
-  useRawData(source, Skia.MakeImageFromEncoded);
```

Comment:
do we need to update the signature of useTypeface, useFont, useSVG and so on?

Reply:
Added onError callback to useSvg, useTypeface and useFont :)